_posts/2022-09-27-building-container-images.adoc

```diff
@@ -0,0 +1,173 @@
+---
+title: Building Containers in Rootless Mode on OpenShift
+layout: post
+author: Mario Loriedo
+description: >-
+  In this blog post, we are going to see how to configure Eclispe Che to run `podman build` from a remote development environment.
+categories: []
+keywords: ['buildah', 'podman']
+slug: /@mloriedo/building-container-images
+---
+
+image::/assets/img/building-container-images/che-inner-loop-optimized.png[Developer inner loop including a container build]
+{nbsp} +
+{nbsp} +
+Nowadays the typical developer flow, the inner-loop, includes quite often the build of a `Dockerfile`. Despite that Eclipse Che did not support building container images on OpenShift. That is because it involved granting permissive OpenShift privileges to Che users and we did not want to require that.
```

Comment:
```suggestion
Nowadays the typical developer flow, the inner-loop, quite often includes the building of a `Dockerfile`. Despite that Eclipse Che did not support building container images on OpenShift. That is because it involved granting permissive OpenShift privileges to Che users and we did not want to require that.
```